ReactiveCocoa/Swift/Signal.swift

```diff
@@ -154,6 +154,15 @@ public final class Signal<Value, Error: ErrorType> {
 	}
 }
 
+extension Signal {
+	/// Number of observers observing the signal or `nil` if signal is terminated.
+	///
+	/// Property used for Unit Tests.
+	internal var observersCount: Int? {
```

Comment:
Maybe replace it with something like
```
internal var isObserved: Bool {
  return atomicObservers.value?.count > 0
}
```
It will be enough for the tests and exposes less details. What do you think?